File: API/Models/Producent.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Models
{
    [Table("Producent")]
    public class Producent
    {
        [Key]
        [Column("ID")]
        public int ID { get; set; }

        [Column("Naziv")]
        [MaxLength(255)]
        public string Naziv { get; set; }
        
        
        
        [Column("GodinaRodjenja")]
        [MaxLength(4)]
        public int GodinaRodjenja { get; set; }

        [Column("BrojFilmova")]
        public int BrojFilmova { get; set; }

        [Column("BrojFilmovaNaStanju")]
        public int BrojFilmovaNaStanju { get; set; }

        //public Katalog Katalog { get; set; }
    }
}

File: Producent.js
export class Producent {
    constructor (id, ime, brojfilmova, godinarodj) {
        this.id = id;
        this.ime = ime;
        this.brojFilmova = brojfilmova;
        this.godinaRodjenja=godinarodj; 
        this.brojFilmovaNaStanju = 0;
    }

    updateProducent(plusminus)
    {
        if(plusminus == 1)
            this.brojFilmovaNaStanju++;
        else
            this.brojFilmovaNaStanju--;
    }
}
